test/e2e/dedicated_reader_test.go

```diff
@@ -0,0 +1,74 @@
+//+build e2e
+
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	_ "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var _ = Describe("Openshift on Azure dedicated-cluster-reader e2e tests [DedicatedAdmin]", func() {
+	defer GinkgoRecover()
+
+	BeforeEach(func() {
+		namespace := nameGen.generate("e2e-test-")
```

Comment:
Same comment as with the dedicated admin ns creation/deletion above; move this only inside the test that actually needs it.

Reply:
done